postgreSQL-case-study.md

```diff
@@ -0,0 +1,172 @@
+
+PostgreSQL has a large list of features which consists of many relating to performance, security, programming extensions, and configurations.
+
+Here are a few key features:
+* Runs on most operating systems like Windows, Linux on all recent distributions, macOS, NetBSD, FreeBSD etc.
+* PostgreSQL also supports a lot of data types;
+     Besides the string, numeric, boolean, date, POSTGRES also supports enumerated, geometric, hstores, network adresses, binary, xml, json, array, uuid, monetary and more. 
+* Table inheritance.
+     Allows you to create a child table based on another table, where the child table will include all of the columns in the parent table.
+* Asynchronous replication
+     Asynchronous is the standard in PostgreSQL to replicate data. It's a reliable and easy way to distribute data making setups more failsafe. 
+* Multi-version concurrency control(MVCC)
+    MVCC is an advanced technique for improving database performance in a multi-user environment.
+* Point in time recovery(PITR)
+* Write-ahead logging(WAL)
+* Full support for foreign keys, joins, views, triggers, and stored procedures, in many languages.
+* Supports storage of binary large objects(blob) including pictures, sounds, and videos.
+* Full-text search
+
+Read more: https://www.postgresql.org/about/featurematrix/
+
+## Strengths
+
+### Open-source
+PostgreSQL is an open-source database making it free to use which allows users to use, modify and implement as they want. Thus it has a strong and active community that has contributed to its development. This type of community-driven development has resulted in a lot of participation from its users. 
+
+### Programming language support
+In addition to being a free and open-source software, it has support for writing functions(programming languages) such as:
+    SQL, Python, Perl, Tcl, Java, R, Javascript, shell, and Lua.
+
+### Extensibility
+PostgreSQL is designed to be extensible. It allows you to define your own data types, index types, operators, functional languages, access methods etc, making PostgreSQL extremely programmable.  
+
+### Scalability
+Scalability is how a system/database can handle a growing amount of demands by adding resources, and PostgreSQL can do just that. It can handle large amounts of data and it scales up as the data grows. 
+
+### ACID Compliant
+PostgreSQL is reliable because it is ACID(Atomicity, Consistency, Isolation, and Durability) compliant meaning queries will maintain data integrity. POSTGRES is known for its rock-solid referential and transactional integrity. ACID in a database system refers to four properties of a transaction. These sets of properties make sure a database transaction are processed reliably before and after. Transactions in this case refer to a set of actions/operations that are used to perform some logical set of work that accesses and can possibly modify the contents of a database. We can access the transactions using read and write operations. 
```

Comment:
```suggestion
PostgreSQL is reliable because it is ACID (Atomicity, Consistency, Isolation, and Durability) compliant, meaning queries will maintain data integrity. POSTGRES is known for its rock-solid referential and transactional integrity. ACID in a database system refers to four properties of a transaction. These properties make sure database transactions are processed reliably before and after. Transactions, in this case, refer to a set of actions/operations used to access or modify the contents of a database, such as complex read and write operations. 
```